### PR TITLE
Releasenotes inter app

### PR DESCRIPTION
**Pull request by janstaelensskyline**

### CODE REVIEW

**Review by SkylineThomasCR — APPROVED** [review]

File: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+This change allows our library to dynamically select a more efficient underlying communication protocol if you have a high enough DataMiner version, or to continue using SLNet Subscriptions as a fallback.
+
+>[!NOTE]
+> The DIS Validator should help report possible invalid code when updating to the new InterApp branch.
+
+#### Breaking change - isLegacyDestination removed
+
+The *Send* method has removed the *isLegacyDestination* argument. This argument was intended to briefly assist in the move from the obsolete *SLC.Lib.* namespaces to the current library. This change occurred several years ago, and as such, this argument was dropped to improve library code maintainability.
+
+#### New feature - Enhanced efficiency on InterApp reply calls
```
Comment (SkylineThomasCR) [inline release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md:25]: should we mention limited reply message size?

**Review by SkylineThomasCR — CHANGES_REQUESTED** [review]

File: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md (hunk 1 above)
Comment (janstaelensskyline) [reply release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md:25]: Perhaps yes, I'll add it

**Review by SkylineThomasCR — APPROVED** [review]

File: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+uid: Skyline_DataMiner_Core_InterAppCalls_Range_1.0
+---
+
+# Skyline DataMiner Core InterAppCalls Range 1.0
+
+> [!NOTE]
+> Range 1.0.x.x is supported as from **DataMiner 10.1.0**. This is a continuation and split from the now obsolete [Class Library](xref:ClassLibrary_Range_1.2).
```
Comment (MariekeGO) [inline release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md:8]:
```suggestion
> Range 1.0.x.x is supported as from **DataMiner 10.1.0**. This is based on the now obsolete [Class Library](xref:ClassLibrary_Range_1.2).
```

File: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#### Breaking change - Replying to an InterApp message
+
+Previously, it was possible to reply to an InterApp call in several ways that involved setting parameter 9000001 (or a similar PID) in some way. Now only a *.Reply* call on the incoming message is allowed. This change allows the library to dynamically select a more efficient underlying communication protocol if a high enough DataMiner version is used or to continue using SLNet subscriptions as a fallback.
+
+> [!NOTE]
+> The DIS Validator should help report possible invalid code when you update to the new InterApp branch.
+
+#### Breaking change - isLegacyDestination removed
+
+The *Send* method has removed the *isLegacyDestination* argument. This argument was intended to briefly assist in the move from the obsolete *SLC.Lib.* NuGet to the current library. This change occurred several years ago, and as such, this argument was dropped to improve library code maintainability.
```
Comment (MariekeGO) [inline release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.0.md:21]:
```suggestion
The *isLegacyDestination* argument is now no longer available for the *Send* method. This argument was intended to briefly assist in the move from the obsolete *SLC.Lib.* NuGet to the current library. This change occurred several years ago, and as such, this argument was dropped to improve library code maintainability.
```

**Pull request merged**